pkg/cloud/services/cloudformation/bootstrap.go

```diff
@@ -233,10 +233,37 @@ func controllersPolicy(accountID, partition string) *iam.PolicyDocument {
 					"iam:PassRole",
 				},
 			},
+			{
+				Effect:   iam.EffectAllow,
+				Resource: iam.Resources{"arn:aws:secretsmanager:*:*:secret:aws.cluster.x-k8s.io/*"},
+				Action: iam.Actions{
+					"secretsmanager:CreateSecret",
+					"secretsmanager:DeleteSecret",
+				},
+			},
 		},
 	}
 }
 
+func controlplaneBootstrapSecretPolicy() iam.StatementEntry {
+	return iam.StatementEntry{
+		Effect:   iam.EffectAllow,
+		Resource: iam.Resources{"arn:aws:secretsmanager:*:*:secret:aws.cluster.x-k8s.io/*"},
+		Action: iam.Actions{
+			"secretsmanager:DeleteSecret",
+			"secretsmanager:GetSecretValue",
+		},
+	}
+}
+
+func controlPlanePolicy() *iam.PolicyDocument {
+	policyDocument := cloudProviderControlPlaneAwsPolicy()
+	policyDocument.Statement = append(
+		policyDocument.Statement, controlplaneBootstrapSecretPolicy(),
+	)
+	return policyDocument
+}
+
```

Comment:
If we do decide to keep the code as it currently stands, and default all instances to use encryption (and not only control plane machines), here's your reminder to update the worker node policy too.